api/config/config.go

```diff
@@ -0,0 +1,136 @@
+package config
+
+import (
```

Comment:
It's recommended to avoid using dot imports as they can make the code harder to understand by obscuring what packages identifiers come from.
```suggestion
import (
	"github.com/PaloAltoNetworks/pango"
	ldap "github.com/go-ldap/ldap/v3"
	hcl "github.com/hashicorp/hcl/v2"
	hclsimple "github.com/hashicorp/hcl/v2/hclsimple"
	gorm "github.com/powerhome/go-blocker/api/repository/gorm"
	cty "github.com/zclconf/go-cty/cty"
)
```